CMGTools/TTHAnalysis/python/plotter/functions.cc

```diff
@@ -163,5 +174,86 @@ float relax_cut_in_eta_bins(float val, float eta, float eta1, float eta2, float
 
 }
 
+int regroupSignalRegions(int SR){
```

Comment:
rename to  `regroupSignalRegions_RA5` or something similar, as it's clearly specific of multilep alone